website/docs/table-group/data-table/data-table-code.md

```diff
@@ -222,7 +222,33 @@ You can provide `data` property for `<DataTable.Row/>` . It isn't used in the co
 
 :::
 
-## Access to cells
+### Rows merging
```

Comment:
I'd move both row and column merging to "Table cells" and rename it to "merging cells vertically" and "merging cells horizontally"